mobile/lib/shared/views/immich_loading_overlay.dart

```diff
@@ -9,25 +9,25 @@ class ImmichLoadingOverlay extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return ValueListenableBuilder<bool>(
-      valueListenable: ImmichLoadingOverlayController.appLoader.loaderShowingNotifier,
+      valueListenable:
+          ImmichLoadingOverlayController.appLoader.loaderShowingNotifier,
       builder: (context, shouldShow, child) {
-        if (shouldShow) {
-          return const Scaffold(
-            backgroundColor: Colors.black54,
-            body: Center(
-              child: ImmichLoadingIndicator(),
-            ),
-          );
-        } else {
-          return Container();
-        }
+        return shouldShow
+            ? const Scaffold(
+                backgroundColor: Colors.black54,
+                body: Center(
+                  child: ImmichLoadingIndicator(),
+                ),
+              )
+            : const SizedBox();
```

Comment:
I am not used to return `SizedBox` for empty Widget, is it the same as return `Container()`?

Reply:
Learning a lot from you!